admin/syslog-ng/Makefile

```diff
@@ -69,8 +69,9 @@ define Package/syslog-ng/install
 		install-moduleLTLIBRARIES DESTDIR="$(1)"
 	$(INSTALL_DIR) $(1)/etc/init.d
 	$(INSTALL_BIN) ./files/syslog-ng.init $(1)/etc/init.d/syslog-ng
-	$(INSTALL_DIR) $(1)/etc
+	$(INSTALL_DIR) $(1)/etc/syslog-ng.d
 	$(INSTALL_DATA) ./files/syslog-ng.conf $(1)/etc
+	$(INSTALL_DATA) ./files/custom-logs.conf $(1)/etc/syslog-ng.d/
```

Comment:
Do we really need this file? Having an empty syslog-ng.d directory seems good enough to me.